services/movie.py

```diff
@@ -1,11 +1,13 @@
-from django.db.models import QuerySet
+from django.db import transaction
+from django.db.models import QuerySet, Q
 
 from db.models import Movie
 
 
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str = None
```

Comment:
If argument can be `None` you should show it in type hint. Fix everywhere
```suggestion
    genres_ids: list[int] | None = None,
    actors_ids: list[int] | None = None,
    title: str | None = None
```